common/util.go

```diff
@@ -518,3 +519,13 @@ func ParseAccelDevices(devices string, acceleration ffmpeg.Acceleration) ([]stri
 	}
 	return strings.Split(devices, ","), nil
 }
+
+func ParseEthAddr(strJsonKey string) (string, error) {
+	var keyJson map[string]interface{}
```

Comment:
```suggestion
	var keyJson map[string]string
```

Then, you'll not need casting later.

Reply:
Thanks, that was much easier :)